shared/src/main/scala/scalus/uplc/eval/BuiltinCostModel.scala

```diff
@@ -97,7 +97,9 @@ case class BuiltinCostModel(
     orByteString: DefaultCostingFun[ThreeArguments],
     xorByteString: DefaultCostingFun[ThreeArguments],
     complementByteString: DefaultCostingFun[OneArgument],
-    readBit: DefaultCostingFun[TwoArguments]
+    readBit: DefaultCostingFun[TwoArguments],
+    writeBits: WriteBitsCostingFun,
+    replicateByte: DefaultCostingFun[TwoArguments]
```

Comment:
The Costing function is wrong here. It should be `NumBytesCostedAsNumWords` in Plutus:
[NumBytesCostedAsNumWords](https://github.com/IntersectMBO/plutus/blob/832e119989e3d9c49b939ae11f152e46b3314e7f/plutus-core/plutus-core/src/PlutusCore/Default/Builtins.hs#L1991)
It's the same as for `IntegerToByteString`: https://github.com/nau/scalus/blob/89777130f4fe61615848fa5f2eb58de2f0556672/shared/src/main/scala/scalus/uplc/eval/CostModel.scala#L515
Rename and reuse it